src/ts/Detail/Comments.spec.tsx

```diff
@@ -2,17 +2,21 @@ import 'mocha';
 import * as React from 'react';
 import { expect } from 'chai';
 import Comments from './Comments';
-import { shallow } from 'enzyme';
+import { mountWithTheme } from '../helpers/test-theme';
 
 describe('<Comments />', () => {
-  it('check for inner components: comments, rate, author', () => {
-    const wrapper = shallow(<Comments />);
+  it('check for inner components: comments, rate, author', function(done) {
+    this.timeout(3000);
```

Comment:
we cannot have non-zero timeouts. They will significantly slow the tests down, and are unreliable. Let's use resolves instead.